test/Infrastructure.IntegrationTest/AdminConsole/Repositories/CollectionRepository/CreateDefaultCollectionsTests.cs

```diff
@@ -0,0 +1,150 @@
+            CreateUserForOrgAsync(userRepository, organizationUserRepository, organization),
+            CreateUserForOrgAsync(userRepository, organizationUserRepository, organization)
+        );
+
+        var affectedOrgUserIds = resulOrganizationUsers.Select(organizationUser => organizationUser.Id);
+        var defaultCollectionName = $"default-name-{organizationId}";
+
+
+        await CreateUsersWithExistingDefaultCollectionsAsync(collectionRepository, organizationId, affectedOrgUserIds, defaultCollectionName, resulOrganizationUsers);
+
+        // Act
+        await collectionRepository.CreateDefaultCollectionsAsync(organizationId, affectedOrgUserIds, defaultCollectionName);
+
+        // Assert
+        await AssertAllUsersHaveOneDefaultCollectionAsync(collectionRepository, resulOrganizationUsers, organizationId);
+
+        await CleanupAsync(organizationRepository, userRepository, organization, resulOrganizationUsers.Select(organizationUser => organizationUser.UserId));
+    }
+
+    private static async Task CreateUsersWithExistingDefaultCollectionsAsync(ICollectionRepository collectionRepository,
+        Guid organizationId, IEnumerable<Guid> affectedOrgUserIds, string defaultCollectionName,
+        OrganizationUser[] resulOrganizationUsers)
+    {
+        await collectionRepository.CreateDefaultCollectionsAsync(organizationId, affectedOrgUserIds, defaultCollectionName);
+
+        await AssertAllUsersHaveOneDefaultCollectionAsync(collectionRepository, resulOrganizationUsers, organizationId);
+    }
+
+    private static async Task AssertAllUsersHaveOneDefaultCollectionAsync(ICollectionRepository collectionRepository,
+        OrganizationUser[] resulOrganizationUsers, Guid organizationId)
+    {
+        foreach (var resulOrganizationUser in resulOrganizationUsers)
+        {
+            var collectionDetails = await collectionRepository.GetManyByUserIdAsync(resulOrganizationUser!.UserId.Value);
+            var defaultCollection = collectionDetails
+                .SingleOrDefault(collectionDetail =>
+                    collectionDetail.OrganizationId == organizationId
+                    && collectionDetail.Type == CollectionType.DefaultUserCollection);
+
+            Assert.NotNull(defaultCollection);
+        }
+    }
+
+    private static async Task<OrganizationUser> CreateUserForOrgAsync(IUserRepository userRepository,
+        IOrganizationUserRepository organizationUserRepository, Organization organization)
+    {
+        var user = await userRepository.CreateAsync(new User
+        {
+            Name = "Test User",
+            Email = $"test+{Guid.NewGuid()}@example.com",
+            ApiKey = "TEST",
+            SecurityStamp = "stamp",
+        });
+
+        var orgUser = await organizationUserRepository.CreateAsync(new OrganizationUser
+        {
+            OrganizationId = organization.Id,
+            UserId = user.Id,
+            Status = OrganizationUserStatusType.Confirmed,
+        });
```

Comment:
Check out the extension methods in [OrganizationTestHelpers](https://github.com/bitwarden/server/blob/main/test/Infrastructure.IntegrationTest/AdminConsole/OrganizationTestHelpers.cs), they may help here.